web_m2x_options/static/src/js/form.js

```diff
@@ -40,17 +40,12 @@ openerp.web_m2x_options = function (instance) {
         },
 
         is_option_set: function(option) {
-            if (_.isUndefined(option)) {
-                return false
-            }
-            var is_string = typeof option === 'string'
-            var is_bool = typeof option === 'boolean'
-            if (is_string) {
-                return option === 'true' || option === 'True'
-            } else if (is_bool) {
-                return option
-            }
-            return false
+            //return option in ('True', 'true', true)
+            return {
+                'true': true,
+                'True': true,
+                true: true,
```

Comment:
This is not doing what you expect. Object keys are never bools, just a `"true"` string.
What you want to do is `return $.inArray(option, [true, "true", "True"])`.